test/extended/authorization/authorization.go

```diff
@@ -727,17 +726,6 @@ var _ = g.Describe("[sig-auth][Feature:OpenShiftAuthorization] authorization", f
 						Namespace: hammerProjectName,
 					},
 				}.run(t)
-				subjectAccessReviewTest{
```

Comment:
I'm not seeing why this worked before and broke now, which precise endpoint is this hitting?